src/core/Tile/Tile.tsx

```diff
@@ -147,15 +147,15 @@ export const Tile = (props: TileProps) => {
       {...rest}
     >
       {thumbnail && (
-        <div className='iui-thumbnail'>
+        <div className='iui-tile-thumbnail'>
           {typeof thumbnail === 'string' ? (
```

Comment:
Order of thumbnail elements are different in html and css, but I guess that does not influence anything